### PR TITLE
API call fix + minor cleanup

### PR DESCRIPTION
**Pull request by theofficetroll**
Sorry, this is bigger than it should be but is mostly little fixes.

Modified seed script to only generate 100 items & made it a bit more robust.

Rewrote webpack because it didn't work as written.

Rewrote some scripts for a production environment.

Fixed routing issues server- and client-side.

For the moment, defaulting photos as that API call will need to be rewritten once I know where the call is going to. This was bundled in with the product API call, which was causing false failures and mangled data.

https://trello.com/c/tui9qmcn

### CODE REVIEW

File: client/src/index.jsx [hunk 1]
```diff
@@ -52,75 +53,33 @@ class ProductModule extends React.Component {
...
-      });
-    });
+      },
+      error: (err) => {
+        console.log('err', err);
+      }
+    })
   }
 
   componentDidMount() {
```
Comment (martinsung00) [inline client/src/index.jsx:79]: Unused f(n) declaration; will this be used later on?

File: server/index.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ var parser = require('body-parser');
 const cors = require('cors');
 
 // router
-var router = require('./routes.js');
+var router = require('./newroutes.js');
```
Comment (martinsung00) [inline server/index.js:11]: Use const declaration.

File: server/index.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ var parser = require('body-parser');
...
 
 // run express server
 var app = express();
```
Comment (martinsung00) [inline server/index.js:14]: Use const declaration.

**Review by martinsung00 — APPROVED** [review]
Good to go Mark! Great job on figuring out the webpack. The seeder script looks great. Just one unused method in the front-end (componentDidMount), which is a lower-priority warning. Additionally, some declaration with Var should be converted to Let or Const. There's no confusing methods or code otherwise.

File: client/src/index.jsx (hunk 1 above)
Comment (theofficetroll) [reply client/src/index.jsx:79]: Legacy code. Left for now since my changes didn't affect it.

File: server/index.js (hunk 2 above)
Comment (theofficetroll) [reply server/index.js:11]: Legacy code. Asked Leslie for clarification on whether these types of changes should be made or left alone when making updates.

File: server/index.js (hunk 3 above)
Comment (theofficetroll) [reply server/index.js:14]: Legacy code. Asked Leslie for clarification on whether these types of changes should be made or left alone when making updates.

**Pull request merged**